src/js/base/logic.js

```diff
@@ -246,6 +246,26 @@ ripe.Ripe.prototype.hasFrame = function(frame) {
     return true;
 };
 
+/**
+ * Determines if the model currently loaded (in case there's one) has the
+ * provided video name available in spec.
+ *
+ * Notice that this call does not assure that a video showing is possible
+ * it only determines that according to model's spec it should be

```

Comment:
"it only determines if according to the model's spec it is possible to get such a video."